Hunting Queries/Microsoft 365 Defender/Lateral Movement/NegoExLogins

```diff
@@ -0,0 +1,13 @@
+id: ab619659-ab7c-4ca4-be0c-ca71a07bf4cd
```

Comment:
This needs replacing with a unique GUID, this one is already in use here: https://github.com/Azure/Azure-Sentinel/blob/46906229919827bffa14211341f52dd68e27ad81/Hunting%20Queries/Microsoft%20365%20Defender/Lateral%20Movement/Account%20brute%20force.yaml